### PR TITLE
Add vehicle + OSCC consensus checking tool

### PR DESCRIPTION
**Pull request by shnewto**
Prior to this commit there was no easy way to verify that OSCC
and the vehicle were on the same page after an install. Usually
validation after install required the joystick commander application
but that provides no indication as to whether the vehicle reports
what we expect it to after sending commands.

After this commit the OSCC repo will have the `consensus.py` program
to verify that OSCC and the vehicle are on the same page or report
otherwise. Ideally this program will be the first step taken after
installing OSCC. It will also help pinpoint issues if the vehicle
provides unexpected or error results if OSCC expected different
information.

### CODE REVIEW

File: utils/consensus/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+## Usage
+
+Connect your computer to OSCC's CAN bus and bring up your socketcan interface
+
+```bash
+sudo ip link set can0 type can bitrate 125000
+sudo ip link set up can0
+```
+
+Run `diagnostics.py --help` for more info.
```
Comment (adrienthebo) [inline utils/consensus/README.md]: This looks like a reference to the old name.

File: utils/consensus/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+- `pip`
+- `python-can` (run `pip install python-can`)
+- `docopt` (run `pip install docopt`)
+
+## Usage
+
+Connect your computer to OSCC's CAN bus and bring up your socketcan interface
+
+```bash
+sudo ip link set can0 type can bitrate 125000
```
Comment (adrienthebo) [inline utils/consensus/README.md]: This bitrate seems a bit low, I though it's normally `500000`. Am I missing something?

File: utils/consensus/README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+1. Send commands to increase or decrease throttle pressure.
+1. Verify just that the wheel position is reported. (This one's unique in that we don't require the
+   wheel positions to expect as a success condition. They shouldn't if the car's in park!.
+1. Disable each OSCC module.
+
+## Dependencies
+
+- `python3`
+- `pip`
+- `python-can` (run `pip install python-can`)
```
Comment (adrienthebo) [inline utils/consensus/README.md]: If python3 is in use, will this require `pip3 install`?

Comment (shnewto) [reply utils/consensus/README.md]: That's a good question. My machine's pip defaults to python3. Running `pip --version` reports python 3.5

File: utils/consensus/consensus.py [hunk 4]
```diff
@@ -0,0 +1,446 @@
...
+        print(
+            Fore.MAGENTA + ' status: ',
+            Style.RESET_ALL,
+            'attempting to disable',
+            module.module_name,
+            'module')
+
+        # Attempt to enable the module parameter. Under the hood, this sends the enable brakes CAN
+        # frame to OSCC/DriveKit over its CAN gateway.
+        self.bus.enable_module(module)
```
Comment (rebpdx) [inline utils/consensus/consensus.py]: I think this should be disable_module, switching it to disable does yield desired behavior of disabling but speed of disable is way faster than the rest of the tests on here.

Comment (shnewto) [reply utils/consensus/consensus.py]: addressed in 2e4e851

Comment (shnewto) [reply utils/consensus/consensus.py]: well kind of addressed. This method should still be calling `self.bus.enable_module` as it's the `enable_module` method. In the `disable_module` method the call to `self.bus.enable_module` was replaced with the expected `self.bus.disable_module`

File: utils/consensus/README.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
+# Consensus
+
+The purpose of this Python 3 program is to provide basic validation that there is the vehicle and
+OSCC can reach a consensus. Ideally it's a method to quickly verify whether your vehicle is in a
```
Comment (cotos) [inline utils/consensus/README.md]: The first sentence leaves me confused

Comment (shnewto) [reply utils/consensus/README.md]: :ok_hand:

**Pull request merged**